### ISSUE TITLE
(can upgrade) Rathalos attack fireball ground, uncontrollable and can fly with ground attacks, galleom missing rush fall move and missile basic attack (in source code)

### ISSUE THREAD

**Issue by RPTOR914** [NemesisSSBU/Playable_Bosses#35]
Those are the issues that I saw for the source code, or the other thing is to try to put the limit code so that ganon and galleom don't die, that's it

To fix the rathalos fly with ground attacks, I changed a variable from grounded = true or false to all rathalos movements (except charge fireball) to start.

Example

 if StatusModule::status_kind(boss_boma) ==*ITEM_LIOLEUSBOSS_STATUS_KIND_ATTACK_BACK_JUMP_FIREBALL {
                            GROUNDED = true
                            if MotionModule::frame(boss_boma) >= MotionModule::end_frame(boss_boma) - 2.0 {
                                CONTROLLABLE = true;
                                FRESH_CONTROL = true;
                            }
                        }

And the galleom is easy (for me) to add

In summary, you can share with me the limit border for general and i try to modify or update your mod with these issues please, (sorry i have a spanish keyboard)

Thanks

**Comment by NemesisSSBU** [comment]
Thank you for reporting these issues and diving this deep into the code. I just committed the new source code changes to GitHub. Galleom now has a new rush (jump button) and his missile attack (right DPAD button).

Now I'm going to look into Rathalos and make some tweaks to him.

**Comment by RPTOR914** [comment]
Ok thanks :)

**Comment by NemesisSSBU** [comment]
Thank you for your very detailed explanation and problem solving. I've gone ahead and implemented it within the [latest update](https://github.com/NemesisSSBU/Playable_Bosses/releases/tag/1.0.0)
